packages/esm-commons-lib/src/index.ts

```diff
@@ -46,4 +46,5 @@ export * from './components/card-summary/summary-card.component';
 export * from './components/expandable-list-widget/expandable-list.component';
 export * from './components/extension-conditional-renderer/patient-based-extension-renderer';
 export * from './components/patient-table/patient-table.component';
-export * from './hooks/useLastEncounter';
\ No newline at end of file
+export * from './hooks/useLastEncounter';
+export * from './styleguide/tiles.scss';
```

Comment:
We don't need this. These exports are for components that can be imported outside this esm for reuse.

Reply:
done